hack/generator/pkg/astmodel/storage_conversion_factories.go

```diff
@@ -0,0 +1,361 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astmodel
+
+import (
+	"fmt"
+	"github.com/Azure/k8s-infra/hack/generator/pkg/astbuilder"
+	"github.com/dave/dst"
+	"github.com/pkg/errors"
+	"go/token"
+)
+
+// createPropertyConversion tries to create a property conversion between the two provided properties, using all of the
+// available conversion functions in priority order to do so.
+func (fn StorageConversionFunction) createPropertyConversion(
```

Comment:
It feels weird to me that this function is on `StorageConversionFunction` but it's defined in a different file than `StorageConversionFunction` is? Maybe the result of a refactoring at some point in the past?

Consider moving this alongside the other functions on `StorageConversionFunction`